typescript-typing-tutor/index.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="styles.css" />
+    <title>Document</title>
+  </head>
+  <body>
+    <div class id="typing-tutor">
+      <p id="phrase">
+        <span>g</span>
+        <span>r</span>
+        <span>u</span>
```

Comment:
You want the first letter to start with the underline so you should just give the first `span` your `current` class.